drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/test/java/org/drools/workbench/screens/guided/dtable/client/widget/table/GuidedDecisionTablePresenterTest.java

```diff
@@ -1426,9 +1449,12 @@ public void onDeleteSelectedCellsWithoutSelections() {
 
     @Test
     public void onDeleteSelectedColumnsWithSelections() throws ModelSynchronizer.MoveColumnVetoException {
-        final AttributeCol52 column = new AttributeCol52() {{
-            setAttribute("attribute1");
-        }};
+        final AttributeCol52 column = new AttributeCol52() {
```

Comment:
Is this Eclipse's formatting of static initialiser blocks (in anonymous sub-classes!)